index.js

```diff
@@ -37,7 +88,7 @@ const modelTemplate = ({ model, columns }) => `"${
     <tr><td bgcolor="lightblue">${model.name}</td></tr>
     ${
       columns
-        ? Object.values(model.attributes)
+        ? Object.values(model.rawAttributes)
```

Comment:
is there a reason for this change? What does it do?

Reply:
This is in response to issue #10, solved by [this comment](https://github.com/zekenie/sequelize-erd/issues/10#issuecomment-525941258). Apparently Sequelize 5 changed something so the values targeted here are accessible via `rawAttributes` instead of `attributes`.